tests/scripts/create-dev-cluster.sh

```diff
@@ -120,16 +131,17 @@ enable_monitoring() {
 
 show_usage() {
     echo ""
-    echo " Usage: $(basename "$0") [-r] [-d /path/to/rook-examples/dir]"
-    echo "  -r        Enable rook orchestrator"
-    echo "  -m        Enable monitoring"
-    echo "  -d value  Path to Rook examples directory (i.e github.com/rook/rook/deploy/examples)"
+    echo " Usage: $(basename "$0") [-r] [-m] [-p <profile-name>] [-d /path/to/rook-examples/dir]"
+    echo "  -r                Enable rook orchestrator"
+    echo "  -m                Enable monitoring"
+    echo "  -p <profile-name> Specify the minikube profile name"
+    echo "  -d value          Path to Rook examples directory (i.e github.com/rook/rook/deploy/examples)"
 }
 
 ####################################################################
 ################# MAIN #############################################
 
-while getopts ":hrmfd:" opt; do
+while getopts ":hrmfd:p:" opt; do
```

Comment:
I'm curious why this is `:hrmfd:p:` and not `:hrmfdp:`

Reply:
@BlaineEXE  the `:`  is how `getopt` knows whether an option needs a value or not. In our case it's only used with  `-p` which requires specifying a profile and `"-d"` which requires a directory.